GitHub/Get-GitHubPullRequest/Get-GitHubPullRequest.psm1

```diff
@@ -91,7 +91,7 @@
 
             $credentialsBase64 = [System.Convert]::ToBase64String([System.Text.Encoding]::ASCII.GetBytes($apiCredentials))
             $webRequestParameters = @{
-                Uri = "https://api.github.com/repos/$repositoryPath/pulls/$pullRequestId"
+                Uri     = "https://api.github.com/repos/$repositoryPath/pulls/$pullRequestId"
```

Comment:
Let's not do this tab stop-alignment anywhere, please.

Reply:
VS code was doing this automatically, because
![image](https://user-images.githubusercontent.com/2797449/208310418-7c0a657f-f631-4324-9bd0-a3b01b47cb7d.png)
I think this was the default, so it felt natural to follow the automatic formatting, but yeah, it's not in our conventions at the moment (although it could be, because: https://github.com/PoshCode/PowerShellPracticeAndStyle/blob/master/Style-Guide/Code-Layout-and-Formatting.md#avoid-using-semicolons--as-line-terminators).
Btw it's aligning the equation signs according to the longest property name, not for the next tab.